### DISCUSSION TITLE
Forms Docs - Independent Action button usage (improvement)

### DISCUSSION THREAD

**Discussion by manper555** [filamentphp/filament#7588] [Ideas]
IMHO for absolutely noobs as I'm will be very time saving if this example were added to docs:

```php
->schema([
    \Filament\Forms\Components\Actions::make([
        \Filament\Forms\Components\Actions\Action::make('myAction')
            ->action(function ($data, $record) {
                // $data will return the submitted form data
                // $record will return the model if you are editing 
            })
    ])
```

It shows the full components path (for not initiated) & how to pass the current record data to be used in the services/controllers/classes of the app.

CREDIT: This snippet was kindly answered in Discord at https://discord.com/channels/883083792112300104/1138486658128421016/1138760642719920208 by toeknee user

**Comment by danharrin** [comment]
Please open a PR to improve the docs then

**Reply by manper555** [reply]
Thanks for your reply and congrats for this amazing product. As I said I'm so noob that don't know how to make a PR properly. So when I learned it, I'll do.

**Reply by danharrin** [reply]
Click the "Edit" button at the bottom of the docs page

**Reply by manper555** [reply]
Guess I've done (or cracked some important...  lol ). Thanks Dan!
